2021-07-21 00:44:18,588 INFO    Thread-1487:29975 [wandb_setup.py:_flush():69] setting env: {'project': 'tess', 'entity': 'parsellsx', 'root_dir': '/home/parsellsx/berkeley-seti', 'run_id': 'nyu4zqp2', 'sweep_param_path': '/home/parsellsx/berkeley-seti/wandb/sweep-0b4ckdoq/config-nyu4zqp2.yaml', 'sweep_id': '0b4ckdoq'}
2021-07-21 00:44:18,588 INFO    Thread-1487:29975 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-21 00:44:18,588 INFO    Thread-1487:29975 [wandb_init.py:_log_setup():337] Logging user logs to /home/parsellsx/berkeley-seti/wandb/run-20210721_004418-nyu4zqp2/logs/debug.log
2021-07-21 00:44:18,588 INFO    Thread-1487:29975 [wandb_init.py:_log_setup():338] Logging internal logs to /home/parsellsx/berkeley-seti/wandb/run-20210721_004418-nyu4zqp2/logs/debug-internal.log
2021-07-21 00:44:18,589 INFO    Thread-1487:29975 [wandb_init.py:init():370] calling init triggers
2021-07-21 00:44:18,589 INFO    Thread-1487:29975 [wandb_init.py:init():377] wandb.init called with sweep_config: {'C': 0.001, 'gamma': 'scale', 'kernel': 'rbf', 'tol': 0.001}
config: {}
2021-07-21 00:44:18,589 INFO    Thread-1487:29975 [wandb_init.py:init():419] starting backend
2021-07-21 00:44:18,589 INFO    Thread-1487:29975 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-21 00:44:18,592 INFO    Thread-1487:29975 [backend.py:ensure_launched():135] starting backend process...
2021-07-21 00:44:18,611 INFO    Thread-1487:29975 [backend.py:ensure_launched():140] started backend process with pid: 24173
2021-07-21 00:44:18,612 INFO    Thread-1487:29975 [wandb_init.py:init():424] backend started and connected
2021-07-21 00:44:18,644 INFO    Thread-1487:29975 [wandb_run.py:_config_callback():872] config_cb None None {'C': 0.001, 'gamma': 'scale', 'kernel': 'rbf', 'tol': 0.001}
2021-07-21 00:44:18,645 INFO    Thread-1487:29975 [wandb_run.py:_label_probe_notebook():844] probe notebook
2021-07-21 00:44:18,645 INFO    Thread-1487:29975 [wandb_run.py:_label_probe_notebook():851] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-21 00:44:18,645 INFO    Thread-1487:29975 [wandb_init.py:init():472] updated telemetry
2021-07-21 00:44:18,646 INFO    Thread-1487:29975 [wandb_init.py:init():491] communicating current version
2021-07-21 00:44:19,298 INFO    Thread-1487:29975 [wandb_init.py:init():496] got version response 
2021-07-21 00:44:19,298 INFO    Thread-1487:29975 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-21 00:44:19,399 INFO    Thread-1487:29975 [wandb_init.py:init():529] starting run threads in backend
2021-07-21 00:44:24,103 ERROR   Thread-1487:29975 [wandb_init.py:init():787] error
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 761, in init
    run = wi.init()
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 532, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 806, in communicate_run_start
    result = self._communicate(rec)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 544, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
Exception
